tools/password/md5_lookup.rb

```diff
@@ -368,11 +368,11 @@ def run
       input = @opts[:input]
       dbs   = @opts[:databases]
 
-      disclamer = Md5LookupUtility::Disclaimer.new
+      disclaimer = Md5LookupUtility::Disclaimer.new
```

Comment:
Not a blocker; the tests seem wrong too

https://github.com/rapid7/metasploit-framework/blob/ee3b8b40b690f6209cd3d8797eb26799e6e07565/spec/tools/md5_lookup_spec.rb#L104-L106